trio/_core/_multierror.py

```diff
@@ -421,6 +420,7 @@ def traceback_exception_init(
                         # copy the set of _seen exceptions so that duplicates
                         # shared between sub-exceptions are not omitted
                         _seen=None if seen_was_none else set(_seen),
+                        **kwargs
```

Comment:
To pass style checks.
```suggestion
                        **kwargs,
```